File: app/src/main/java/com/websarva/wings/android/kaigonote/ZikoActivity.java
package com.websarva.wings.android.kaigonote;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.websarva.wings.android.kaigonote.databinding.ZikoBinding;

public class ZikoActivity extends AppCompatActivity {
    private ZikoBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ZikoBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
    }
}

File: app/src/main/java/com/websarva/wings/android/kaigonote/ZikoActivity.kt
package com.websarva.wings.android.kaigonote

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.websarva.wings.android.kaigonote.databinding.ZikoBinding

class ZikoActivity : AppCompatActivity() {
    private var binding: ZikoBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ZikoBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
    }
}